contracts/src/v0.8/automation/dev/v2_3/AutomationRegistry2_3.sol

```diff
@@ -213,7 +213,7 @@ contract AutomationRegistry2_3 is AutomationRegistryBase2_3, OCR2Abstract, Chain
           emit UpkeepPerformed(
             report.upkeepIds[i],
             upkeepTransmitInfo[i].performSuccess,
-            receipt.gasReimbursementInJuels + receipt.premiumInJuels, // TODO - this is currently the LINK amount, but may change to billing token
+            receipt.gasReimbursementInJuels + receipt.premiumInJuels,
```

Comment:
Should we nuke those 2 receipt related fields given that we have UpkeepCharged now? Not sure if we have backward compatibility issue, needs to confirm.

Reply:
I think:
* we need to keep this field for backwards compatibility (sadly 😞)
* can we change this to be the user's billing token? ex `receipt.gasInBIllingToken + receipt.premiumInBilliingToken`??